### PR TITLE
Bug 1722944: rsyslog log collector dropping logs

### PR DESCRIPTION
**Pull request by richm**
Increase the elasticsearch queue size - the goal is that the limit
should come from the available disk space, not the number of
records allowed in the queue.
Improve the stats queue so that it does not interfere with the
other outputs.
Label all actions - in conjunction with https://github.com/openshift/origin-aggregated-logging/pull/1713
will greatly improve stats legibility

### CODE REVIEW

**Comment by richm** [comment]
ok - ready to review

File: files/rsyslog/55-elasticsearch.conf [hunk 1]
```diff
@@ -62,9 +62,10 @@ ruleset(name="try_es") {
             queue.maxFileSize=`echo $ES_QUEUE_MAXFILESIZE`
             queue.checkpointInterval=`echo $ES_QUEUE_CHECKPOINTINTERVAL`
             queue.syncqueuefiles="on"
+            queue.size=`echo $ES_QUEUE_SIZE`
             action.resumeRetryCount="-1"
             )
-            # not yet supported - action.resumeIntervalMax=`echo $ES_RETRY_WAIT`
+            # not yet supported - RHEL 8.2 rebase - action.resumeIntervalMax=`echo $ES_RETRY_WAIT`
```
Comment (nhosoi) [inline files/rsyslog/55-elasticsearch.conf]: Out of curiosity... OAL vendored_src does not contain the latest rsyslog source codes?

Comment (richm) [reply files/rsyslog/55-elasticsearch.conf]: it contains the latest rsyslog source code from rhel8

Comment (nhosoi) [reply files/rsyslog/55-elasticsearch.conf]: Then, we could uncomment???

Comment (richm) [reply files/rsyslog/55-elasticsearch.conf]: rhel 8.2 has not been released yet . . .

**Pull request merged**